kubernetes/secrets/config/code-hosts.json.erb

```diff
@@ -0,0 +1,19 @@
+{
+  "GITHUB": [
+    {
+      "url": "https://github.com",
+      "token": "<%= github_token %>",
+      "repositoryQuery": [
+        "org:ocf is:public archived:false"
+      ],
+      "exclude": [
+        {
+          "name": "ocf/apache2-suexec-ocf"
+        },
+        {
+          "name": "ocf/octocatalog-diff"
```

Comment:
why tho?

Reply:
This (and `ocf/apache2-suexec-ocf` above it) are both excluded because they contain a full copy of the upstream debian package with only a few OCF-specific patches on top of them ([[1]](https://github.com/ocf/apache2-suexec-ocf/blob/ocf/2.4.25-3/debian/patches/ocf-suexec.patch) and [[2]](https://github.com/ocf/apache2-suexec-ocf/blob/ocf/2.4.25-3/debian/patches/ocf_build_suexec.patch) for `apache2-suexec-ocf` and [[1]](https://github.com/ocf/octocatalog-diff/blob/ocf/1.5.4-1ocf3/debian/patches/ocf-puppet-6.patch) for `octocatalog-diff`). Most of the search results including them end up matching code that's not actually written or maintained by OCF members.

The same is true for files like `vendor/venv-update` occasionally and it'd be great to exclude that too, but it's definitely less of a problem than a whole repo matching.